### ISSUE TITLE
[data grid] Aggregation is not accessible with keyboard in the column menu

### ISSUE THREAD

**Issue by joserodolfofreitas** [mui/mui-x#7216]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Steps to reproduce 🕹

Link to live example:

Steps:
1. Visit one of the demos at https://mui.com/x/react-data-grid/aggregation/
2. Enter a column menu presenting the aggregation select field
3. try navigating the menu and changing the aggregation select.

### Current behavior 😯

You can reach the aggregation text field, but you cannot get `focus` on the select input to change its value.

### Expected behavior 🤔

It should be possible to change the aggregation function with keyboards only.

**Comment by annawatson-wk** [comment]
Hi! We have a Data Grid Premium license and we're running into this same issue. Do you have any updates on a fix?
